src/shared-components/alert/index.tsx

```diff
@@ -114,77 +102,65 @@ export class Alert extends React.Component<AlertProps, AlertState> {
       return;
     }
 
-    this.timer = window.setTimeout(
-      this.alertExitHandler,
+    timer = window.setTimeout(
+      alertExitHandler,
       Number(duration) * 1000 - ANIMATION_DELAY,
     );
-  }
 
-  componentWillUnmount() {
-    if (this.timer) {
-      window.clearTimeout(this.timer);
-    }
-  }
-
-  alertExitHandler = () => {
-    const { onExit, ...rest } = this.props;
-    this.setState({ exiting: true });
-
-    window.clearTimeout(this.timer);
-
-    window.setTimeout(() => {
-      this.setState({ exited: true });
+    // ComponentWillUnmount cleanup
```

Comment:
We can remove this note--we'll have this repo fully converted over so I don't think we don't need to keep referencing class components.

Reply:
done